### PR TITLE
added `storedValues` to new features

### PR DESCRIPTION
**Pull request by jsteemann**
Docs PR for https://github.com/arangodb/arangodb/pull/15416

### CODE REVIEW

File: 3.10/indexing-working-with-indexes.md [hunk 1]
```diff
@@ -82,34 +82,59 @@ Note that `_key` implicitly has an index assigned to it.
...
+
+**storedValues**: in indexes of type *persistent*, additional attributes can be
+stored in the index. These additional attributes cannot be used for
+index lookups or for sorting, but they can be used for projections. This allows an
+index to fully cover more queries and avoid extra document lookups.
+The maximum number of attributes in **storedValues** is 32.
+It is not possible to create multiple indexes with the same **fields** attributes
+and uniqueness but different **storedValues** attributes. That means the value of 
+**storedValues** is not considered by index creation calls when checking if an 
+index is already present or needs to be created.
```
Comment (Dronplane) [inline 3.10/indexing-working-with-indexes.md:121]: Suggest to add how we treat missing attributes. And mention that `storedValues` are not checked for uniqueness if `unique` is set. Just to be more explicit for end-user.

Comment (jsteemann) [reply 3.10/indexing-working-with-indexes.md:121]: I pushed a change that addresses this.

**Review by ansoboleva — APPROVED** [review]
reviewed

**Review by Dronplane — APPROVED** [review]
LGTM

**Pull request merged**